prefect_aws/batch.py

```diff
@@ -24,7 +24,7 @@ async def batch_submit(
         job_definition: The AWS batch job definition.
         job_queue: Name of the AWS batch job queue.
         aws_credentials: Credentials to use for authentication with AWS.
-        batch_kwargs: Additional keyword arguments to pass to the boto3
+        **batch_kwargs: Additional keyword arguments to pass to the boto3
```

Comment:
This would be a breaking change. Are we sure that we want to make this change?

Reply:
It's actually not breaking, it was originally **, just not documented